docs/tutorials/writing-sd-card-image-etcher.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Writing an SD Card Image Using Etcher (Cross-Platform)
+subject: SD Card Image
+---
+
+1. Download and install Etcher from [their website](http://etcher.io).
+2. Launch Etcher after it has finished installing.
+
+   <div>{% include screenshot.html source="/images/etcher/home.png" caption="Etcher home screen" %}</div>
+
+3. Click the "Select image" button and browse to the folder where you
+   downloaded an ev3dev release. Select the file that you downloaded.
+   The release can be a `.img.zip`, `.img.xz` or just `.img`; whichever
```

Comment:
Again, if we just have the `.xz` file above, then we can simplify this.

We can leave out the `.img` file. We will never publish a raw image. It would be a huge download.